### PR TITLE
Fix doc

### LINKED ISSUE #7481: [doc]Official website document information error
In the official website tutorial, Under this directory ---Using Ray / AsyncIO / Concurrency for Actors / Async Actor,When introducing the 'max_concurrency' flag, in the sample code below it is 'max_concurreny', The 'max_concurreny' missing one letter

### PR DESCRIPTION
**Pull request by allenanswerzq**
## Why are these changes needed?

Fix small documentation issues.

## Related issue number

Closes #7481  #7281 

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://ray.readthedocs.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

File: doc/source/rllib-algorithms.rst [hunk 1]
```diff
@@ -1,6 +1,28 @@
 RLlib Algorithms
 ================
 
+Feature Compatibility Matrix
```
Comment (ericl) [inline doc/source/rllib-algorithms.rst]: I actually just moved this in https://github.com/ray-project/ray/pull/7503/files

Can you rebase the PR?

File: doc/source/async_api.rst [hunk 2]
```diff
@@ -105,7 +105,7 @@ You can limit the number of concurrent task running at once using the
             await asyncio.sleep(1) # Network, I/O task here
             print("ended")
 
-    actor = AsyncActor.options(max_concurreny=10).remote()
+    actor = AsyncActor.options(max_concurrency=10).remote()
```
Comment (ericl) [inline doc/source/async_api.rst]: 👍

**Comment by landcold7** [comment]
rebase done, :-)

File: .gitignore [hunk 3]
```diff
@@ -173,3 +173,6 @@ tools/prometheus*
 
 # ray project files
 project-id
+
+# Cython example files
+doc/examples/cython/**/*
```
Comment (edoakes) [inline .gitignore]: I don't think this should be ignored - is there a reason you added it?

Comment (landcold7) [reply .gitignore]: These files are newly generated after build ray on my pc, and they didn't add into the git system. so I added just for my own convenience.

**Comment by landcold7** [comment]
Fixed.

**Review by edoakes — APPROVED** [review]
Thanks for the fix!

**Pull request merged**